sections/images.liquid

```diff
@@ -354,7 +354,7 @@
                         {%- endif -%}
 
                         {%- if description != blank -%}
-                          <div class="images__description text-medium bq-content rx-content">{{- description | strip_html | truncate: 150, " ..." -}}</div>
+                          <div class="images__description text-medium bq-content rx-content">{{- description -}}</div>
```

Comment:
Is this `description` sanitized to be free of malicious scripting? 
(it would be cool if we have `strip_unsafe_html` filter)

Reply:
This HTML is controlled by the shop owner I'm not sure if it needs to be sanitised. We plan to open theme code editor to the public anyway so they will be able to add what ever they want to the code anyway.

Reply:
> We plan to open theme code editor to the public anyway so they will be able to add what ever they want to the code anyway.

OK, let's hope that ["no bad people will show up"](https://www.youtube.com/watch?v=eBShN8qT4lk)